app/src/newdocs/content/streams.mdx

```diff
@@ -19,7 +33,61 @@ import docsStyles from '$newdocs/components/DocsLayout/docsLayout.pcss'
 <ScrollableAnchor id="work-with-streams-via-sdks"><div>
 
 ## Work with streams via SDKs
---content--
+
+The easiest way to work with streams is to use the JavaScript client, which works in `node.js` as well as the browser. See the <Link to={links.newdocs.SDKs}>SDKs section</Link> for the official and community-maintained client libraries available for various languages:
+
+- **JavaScript** client works in the browser as well as node.js.
+- **Java** client is work-in-progress but already implements the most important set of functionality.
+
+Client libraries for other languages are on the <a href={links.community.trello} target="_blank" rel="nofollow noopener noreferrer">roadmap</a>. If there isn't a client library available for your language, you can dive into the details of the [websocket protocol](https://github.com/streamr-dev/streamr-client-protocol-js/blob/master/PROTOCOL.md) 
+
+If you'd like to contribute a client library and get it listed here, please get in touch on [Telegram](https://t.me/streamrdata), [Reddit](https://www.reddit.com/r/streamr/) or [email](mailto:contact@streamr.com)!
+
+### Authentication
+
+When reading from or writing to Streams, you need to provide a session token or an API key, or login with an Ethereum account. To read more about API keys or obtaining a session token, refer to the <Link to={links.newdocs.api}>Authentication section of the API documentation</Link>.
+
+### Creating a client instance with API key
+
+**Javascript**
+
+<CodeSnippet language='javascript'>{CreateJavascriptClient}</CodeSnippet>
+
+Authenticating with an Ethereum private key contained in an Ethereum (web3) provider:
+
+<CodeSnippet language='javascript'>{AuthJavascriptClient}</CodeSnippet>
+
+**Java**
+
+<CodeSnippet language='java'>{CreateJavaClient}</CodeSnippet>
+
+To authenticate with an Ethereum account, a private key must be provided. The library will automatically initiate a challenge-response protocol to allow you to prove that you own the Ethereum private key without revealing it. You will be identified with the associated Ethereum public address.
+
+<CodeSnippet language='java'>{AuthJavaClient}</CodeSnippet>
+
```

Comment:
Perhaps should drop all of this auth content and put it in the auth section i.e. there's more about auth here than actually reading/writing to streams.

Reply:
Also note this content is near identical to what's added in https://github.com/streamr-dev/streamr-platform/pull/557/files#diff-923b7bd635c3f3e482653871e4d467f4 so probably should think about what belongs where.

IMO this section should maybe be a "deeper" dive into the intro content linked above, so perhaps it covers things like resends & realtime vs historical data? Error handling? rate limiting? etc

Reply:
Ok, I left out the authentication and kept only the simple examples here. I feel the deep dive should happen in the SDK section (or leave it for the Github repos). But maybe those examples could be added here as well.